### ISSUE TITLE
Headings within code blocks appear in ToC

### ISSUE THREAD

**Issue by danieldilly** [Requarks/wiki#1596]
**Describe the bug**
If you create a code block with a heading element in your markdown file like this:

```
<h1>Sample code</h1>
```

It will show up in the Table of Contents.

**To Reproduce**
1. Create a new page of type Markdown
2. Create a new code block with a heading tag within
3. View the rendered table of contents

**Expected behavior**
I don't believe headings within code blocks should appear within the Table of Contents.  If anything, have it optional with the default set to `false`.

**Comment by Smankusors** [comment]
do you mean this one?

    ```
    <h1>Sample code</h1>
    ```

instead of this?

```
<h1>Sample code</h1>
```

**Comment by NGPixel** [comment]
Fixed by b529ad21c97a7093e9ecc35e2e72fb8148f0c9d0
